mappings/net/minecraft/world/gen/chunk/ChunkGeneratorConfig.mapping

```diff
@@ -19,6 +19,8 @@ CLASS net/minecraft/class_2888 net/minecraft/world/gen/chunk/ChunkGeneratorConfi
 	FIELD field_23987 netherStructureSpacing I
 	FIELD field_23988 netherStructureSeparation I
 	FIELD field_23989 netherStructureSeedModifier I
+	FIELD field_24507 ruinedPortalSpacing I
+	FIELD field_24508 ruinedPortalSeperation I
```

Comment:
seperation isn't a real word. you want `separation`

Reply:
fixed